maths/monte_carlo.py

```diff
@@ -35,34 +36,43 @@ def in_circle(x: float, y: float) -> bool:
     print("The total error is ", abs(pi - pi_estimate))
 
 
-def area_under_line_estimator(iterations: int,
+def area_under_curve_estimator(iterations: int,
+                              function_to_integrate: Callable[[float], float],
```

Comment:
`Callable[[float], float]` ?  I would have thought `Callable[float, float]`.  What does the extra bracket do for us?

Reply:
Thanks. I tried removing bracket. It gives following error. I think it expects a list.

TypeError: Callable[args, result]: args must be a list. Got <class 'float'>